style-about.css

```diff
@@ -0,0 +1,230 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;  
+}
+
+
+.header { 
+  font-family: 'Space Mono', sans-serif;
+  margin-bottom: 0;
+  font-weight: 700;
+  font-size: 48px;
+  letter-spacing: -2.14286px;
```

Comment:
letter-spacing should be relative to font-size. should use em or %. if you want to use em: `letter-spacing: -0.021em`

Reply:
fixed